system/jlib/jtrace.cpp

```diff
@@ -348,7 +349,7 @@ class CSpan : public CInterfaceOf<ISpan>
 
             StringBuffer out;
             toString(out);
```

Comment:
Worth introducing a new toLog() which logs a subset of the information to avoid duplication.